solr/core/src/java/org/apache/solr/util/PackageTool.java

```diff
@@ -261,44 +261,44 @@ protected void runImpl(CommandLine cli) throws Exception {
         .argName("URL")
         .hasArg()
         .required(true)
-        .desc("Address of the Solr Web application, defaults to: " + SolrCLI.DEFAULT_SOLR_URL)
+        .desc("Address of the Solr Web application, defaults to: " + SolrCLI.DEFAULT_SOLR_URL + '.')
         .build(),
 
         Option.builder("collections")
         .argName("COLLECTIONS")
         .hasArg()
         .required(false)
-        .desc("List of collections. Run './solr package help' for more details.")
+        .desc("List of collections.")
         .build(),
 
         Option.builder("cluster")
         .required(false)
-        .desc("Needed to install cluster level plugins in a package. Run './solr package help' for more details.")
+        .desc("Needed to install cluster level plugins in a package.")
```

Comment:
I'm not sure what this description means. There is no verb?

Reply:
Take a look at what I put in now.    Also, now I see we sometimes say "package" and other times "plugin", I wonder wha tthe right term is?  I *think* it's "package".